appengine/standard/django/pytest.ini

```diff
@@ -0,0 +1,2 @@
+[pytest]
+DJANGO_SETTINGS_MODULE=mysite.settings
```

Comment:
Ah I see. The noxfile in `appengine/standard` doesn't use `noxfile_config.py`. I'll take care of this with another PR.